acra/security-controls/tokenisation/_index.md

```diff
@@ -2,3 +2,170 @@
 title: Tokenisation
 bookCollapseSection: true
 ---
+
+# Tokenization
+
+## Description
+
+Tokenization is a way to transform some sensitive data into pseudo-random representation of same data type.
+A number will remain a number, string will be string, email will be email.
+But the tokenized version will look like a random value.
+Better see than read:
+
+* `789` -> `156749362`
+* `testing testing` -> `lwcIvM3zaJN9cbN`
+* `kate@example.com` -> `sVAOfOwF@mSuGPog.et`
```

Comment:
It seems that email is not data type. I guess that tokenization targets not only data type but also format (email will be email). Should we elaborate on this in phrase above?